operations/alloy-mixin/alerts/opentelemetry.libsonnet

```diff
@@ -22,19 +26,55 @@ local alert = import './utils/alert.jsonnet';
         alert.newRule(
           'OtelcolReceiverRefusedSpans',
           successRateQuery(enableK8sCluster, "otelcol_receiver_refused_spans_total", "otelcol_receiver_accepted_spans_total"),
-          'The receiver pushing spans to the pipeline success rate is below 95%.',
+          'The receiver pushing spans to the pipeline success rate is below %s.' % successThresholdText,
           'The receiver could not push some spans to the pipeline under job {{ $labels.job }}. This could be due to reaching a limit such as the ones imposed by otelcol.processor.memory_limiter.',
-          '10m',
+          pendingPeriod,
+        ),
+
+        // Metrics receiver alerts
+        alert.newRule(
+          'OtelcolReceiverRefusedMetrics',
+          successRateQuery(enableK8sCluster, "otelcol_receiver_refused_metric_points_total", "otelcol_receiver_accepted_metric_points_total"),
+          'The receiver pushing metrics to the pipeline success rate is below %s.' % successThresholdText,
+          'The receiver could not push some metric points to the pipeline under job {{ $labels.job }}. This could be due to reaching a limit such as the ones imposed by otelcol.processor.memory_limiter.',
+          pendingPeriod,
+        ),
+
+        // Logs receiver alerts
+        alert.newRule(
+          'OtelcolReceiverRefusedLogs',
+          successRateQuery(enableK8sCluster, "otelcol_receiver_refused_log_records_total", "otelcol_receiver_accepted_log_records_total"),
+          'The receiver pushing logs to the pipeline success rate is below %s.' % successThresholdText,
+          'The receiver could not push some log records to the pipeline under job {{ $labels.job }}. This could be due to reaching a limit such as the ones imposed by otelcol.processor.memory_limiter.',
+          pendingPeriod,
         ),
 
-        // The exporter success rate is below 95%.
+        // The exporter success rate is below threshold.
         // There could be an issue with the payload or with the destination endpoint.
         alert.newRule(
           'OtelcolExporterFailedSpans',
           successRateQuery(enableK8sCluster, "otelcol_exporter_send_failed_spans_total", "otelcol_exporter_sent_spans_total"),
-          'The exporter sending spans success rate is below 95%.',
+          'The exporter sending spans success rate is below %s.' % successThresholdText,
           'The exporter failed to send spans to their destination under job {{ $labels.job }}. There could be an issue with the payload or with the destination endpoint.',
-          '10m',
+          pendingPeriod,
+        ),
+
+        // Metrics exporter alerts
+        alert.newRule(
+          'OtelcolExporterFailedMetrics',
```

Comment:
With the new prom changes will this immediately fire?

Reply:
I don't think it will. In our internal environments the metric aggregation needs tweaking to not aggregate these by instance label. But we update the mixin manually there, so this change won't trigger it.